gateway/src/main/java/com/lyft/data/gateway/config/RequestRouterConfiguration.java

```diff
@@ -13,5 +13,10 @@
   // Cache dir to store query id <-> backend mapping
   private String cacheDir;
 
+  // Use SSL?
+  private String ssl;
+  private String keystorePath;
```

Comment:
[Dropwizard](https://www.dropwizard.io/1.3.13/docs/manual/configuration.html) lets you configure `SSL`. Though here we are using jetty based app, we could still use DW config to fill these values.
```
applicationConnectors:
    - type: http
      port: 8080
    - type: https
      port: 8443
      keyStorePath: example.keystore
      keyStorePassword: example
      validateCerts: false
```